metricbeat/module/windows/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- key: windows
+  title: "Windows"
+  description: >
```

Comment:
Please mark this as beta: https://github.com/elastic/beats/blob/master/metricbeat/module/jolokia/_meta/fields.yml#L4